File: Java/1006 - Media 2.java
import java.io.IOException;
import java.util.Scanner;

class Main {
    public static void main(String[] args){

        Scanner input = new Scanner(System.in);

        double nt1 = input.nextDouble();
        double nt2 = input.nextDouble();
        double nt3 = input.nextDouble();
    
        double media = (nt1*2 + nt2*3 + nt3*5) / 10;

        System.out.printf("MEDIA = %.1f\n", media);
    }
}

File: C/1006 - Media 2.c
#include <stdio.h>

int main(){

	double nt1, nt2, nt3, media;

	scanf("%lf",&nt1);
	scanf("%lf",&nt2);
	scanf("%lf",&nt3);

	media = (nt1*2 + nt2*3 + nt3*5) / 10;

	printf("MEDIA = %.1lf\n", media);

	return 0;
}